scripts/globals/utils.lua

```diff
@@ -1234,3 +1234,14 @@ function utils.drawIn(target, table)
     target:setLocalVar('[Draw-In]WaitTime', 0)
     return false
 end
+
+function utils.defaultValue(inputValue, defaultValue)
```

Comment:
`valueIfNil` or `defaultIfNil` is better now that I see it, sorry for the runaround